doc/develop/test/bsim.rst

```diff
@@ -76,3 +76,103 @@ Check :ref:`the page on coverage generation <coverage_posix>` for more info.
 
 .. _EDTT:
    https://github.com/EDTTool/EDTT
+
+Tests with radio activity
+*************************
+
+The ``zephyr/tests/bsim/`` folder contains tests meant to be run with BabbleSim's physical layer
+simulation, and therefore cannot be run directly from twister.
+
+The ``compile.sh`` and ``run_parallel.sh`` scripts contained in that folder are used by the CI system to
+build the needed images and execute these tests in batch.
+
+Building and running
+====================
+
+See the :ref:`nrf52_bsim` page for setting up the simulator.
+
+The scripts also expect a few environment variables to be set.
+For example, from Zephyr's root folder, you can run:
+
+.. code-block:: bash
+
+   # Build all the tests
+   WORK_DIR=${ZEPHYR_BASE}/bsim_out ${ZEPHYR_BASE}/tests/bsim/compile.sh
+
+   # Run them (in parallel)
+   RESULTS_FILE=${ZEPHYR_BASE}/myresults.xml \
+      SEARCH_PATH=${ZEPHYR_BASE}/tests/bsim \
+         ${ZEPHYR_BASE}/tests/bsim/run_parallel.sh
+
+Or to run only a specific subset, e.g. host advertising tests:
+
+.. code-block:: bash
+
+   # Build the Bluetooth host advertising tests
+   WORK_DIR=${ZEPHYR_BASE}/bsim_out \
+      ${ZEPHYR_BASE}/tests/bsim/bluetooth/host/adv/compile.sh
+
+   # Run them (in parallel)
+   RESULTS_FILE=${ZEPHYR_BASE}/myresults.xml \
+      SEARCH_PATH=${ZEPHYR_BASE}/tests/bsim/bluetooth/host/adv \
+         ${ZEPHYR_BASE}/tests/bsim/run_parallel.sh
+
+Check the ``run_parallel.sh`` help for more options and examples on how to use this script to run
+the tests in batch.
+
+After building the tests' required binaries you can run a test directly using its individual test
+script.
+
+For example you can build the required binaries for the networking tests with
+
+.. code-block:: bash
+
+   WORK_DIR=${ZEPHYR_BASE}/bsim_out ${ZEPHYR_BASE}/tests/bsim/net/compile.sh
+
+and then directly run one of the tests:
+
+.. code-block:: bash
+
+   ${ZEPHYR_BASE}/tests/bsim/net/sockets/echo_test/tests_scripts/echo_test_802154.sh
+
```

Comment:
On the reorder you seem to have missed the `Generating coverage reports` section?

Reply:
https://github.com/jori-nordic/zephyr/blob/bsim-testing-rst/doc/develop/test/bsim.rst#L63
It's already here